modular_robot_physical/revolve2/modular_robot_physical/_physical_robot_config.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+import pickle
+from dataclasses import dataclass
+
+from revolve2.modular_robot import ModularRobot
+from revolve2.modular_robot.body.base import ActiveHinge
+
+
+@dataclass
+class PhysicalRobotConfig:
+    """The configuration for running a physical robot."""
+
+    modular_robot: ModularRobot
+    """The Modular Robot Object."""
+    hinge_mapping: dict[ActiveHinge, int]  # which servo is triggered by which hinge.
+    simulation_time: int  # in seconds.
+    control_frequency: int
+    inverse_servos: dict[int, bool] | None = None
+    """Inverse servos according to ActiveHinge objects or pin IDs."""
```

Comment:
I do not undetstand what you mean. Should this not be a dict[ActiveHinge, bool]? What does the int key mean